src/app/rpmostree-builtin-status.cxx

```diff
@@ -302,6 +344,20 @@ print_daemon_state (RPMOSTreeSysroot *sysroot_proxy,
                  update_driver_name, update_driver_sd_unit);
       else
         g_print ("AutomaticUpdates: driven by %s\n", update_driver_name);
+      
+      /* only try to get unit's StatusText if we're on the system bus and if 
+       * `update_driver_sd_unit` is a systemd service unit. */
+      if (bus_type == G_BUS_TYPE_SYSTEM && g_str_has_suffix (update_driver_sd_unit, ".service"))
+        {
+          g_autofree const char *update_driver_status = NULL;
+          if (!get_update_driver_sd_status(sysroot_proxy, update_driver_sd_unit,
```

Comment:
We shouldn't make this a hard error. Let's just warn if we get an error in this case?